src/ArticleList.jsx

```diff
@@ -0,0 +1,26 @@
+import { useState, useEffect } from "react";
+import { getArticles } from '../Utils/api'
+import ArticleCard from './ArticleCard'
+
+function ArticleList() {
+    const [articles, setArticles] = useState([]);
+
+    useEffect(() => {
+        getArticles().then((data) => {
+            setArticles(data.articles)
+            console.log(data.articles)
```

Comment:
We really want to remove unnecessary console.logs before comitting + pushing - when we are creating a PR, this means "This piece of work is done", and in the spirit of code unless it's necessary logs, they should be removed